ruby_playbook

```diff
@@ -0,0 +1,83 @@
+- name: ToDo App
+  hosts: ruby-hosts
+  remote_user: ec2-user
+  vars_prompt:
+
+    - name: RUBY_KEY
+      prompt: What is your API key?
+      private: no
+
+    - name: RUBY_TOKEN
+      prompt: What is your token?
+      private: no
+
+    - name: BOARD_ID
+      prompt: What is the board ID?
+      private: no
+
+    - name: TO_DO_ID
+      prompt: What is the ID of the To-Do list?
+      private: no
+
+    - name: DOING_ID
+      prompt: What is the ID of the Doing list?
+      private: no 
+
+    - name: DONE_ID
+      prompt: What is the ID of the Done list?
+      private: no
+
+
+
+  tasks:
+  - name: Install Git
+    ansible.builtin.yum:
+      name: git
+      state: latest
+    become: yes
+
+  - name: Install Python3
+    ansible.builtin.yum:
+      name: python3
+      state: latest
+
+  - name: Install Poetry
+    ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3
+    args:
+      chdir: ~/.local/bin/
```

Comment:
You don't need to change directory in order to run the command `curl -sSL https://install.python-poetry.org | python3 -`. It doesn't matter where you run it from.

Also, note that you're missing a hyphen at the end. This hyphen is to tell `python3` to read the piped input, rather than opening up a terminal or expecting a filename.